generated/lib/HeaderSelector.php

```diff
@@ -61,6 +61,14 @@ public function selectHeaders(array $accept, string $contentType, bool $isMultip
             $headers['Content-Type'] = $contentType;
         }
 
+        // Add Passage version header
+        try {
+            $packageVersion = \Composer\InstalledVersions::getPrettyVersion('passageidentity/passage-php');
```

Comment:
this function seemed like the best place to put it since it's called in each of the API classes

```
grep -lr 'selectHeaders('

./generated/lib/HeaderSelector.php
./generated/lib/Api/MagicLinksApi.php
./generated/lib/Api/TokensApi.php
./generated/lib/Api/UserDevicesApi.php
./generated/lib/Api/UsersApi.php
./generated/lib/Api/AppsApi.php
./custom/test/PassageTest.php
```

Reply:
Won't this function be deleted when the code is regenerated?

Reply:
Is it an option to set the `artifactVersion` when running the generator? [Docs](https://openapi-generator.tech/docs/generators/php).

Just glanced quickly to see if they had a config option that might help. I'm not sure what `artifact` refers to here.

Reply:
> Won't this function be deleted when the code is regenerated?
Yes, but it gets re-added at the end of the generation script [here](https://github.com/passageidentity/passage-php/pull/69/files#diff-be3b2769f01dd9f7e4dc733ecacac374f506777d575a92280908410bfd86e89eR48)

> Is it an option to set the artifactVersion when running the generator? [Docs](https://openapi-generator.tech/docs/generators/php).
This looks to only be for setting the composer.json version, which wouldn't affect adding a header to our http requests.

Reply:
I was over complicating this in my mind. You ran the generate script already so this is doing the initial merge. In the future this will run every time and the function will be in `HeaderSelector`. 👍